wasm/src/api.rs

```diff
@@ -2,23 +2,35 @@ use wasm_bindgen::prelude::*;
 
 use clvmr::allocator::Allocator;
 use clvmr::chia_dialect::ChiaDialect;
+use clvmr::chia_dialect::NO_NEG_DIV as _no_neg_div;
+use clvmr::chia_dialect::NO_UNKNOWN_OPS as _no_unknown_ops;
 use clvmr::cost::Cost;
 use clvmr::node::Node;
 use clvmr::run_program::run_program;
-use clvmr::serialize::{node_from_bytes, node_to_bytes};
+use clvmr::serialize::{node_from_bytes, node_to_bytes, serialized_length_from_bytes};
 
 // When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
 // allocator.
 #[cfg(feature = "wee_alloc")]
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
-// NOTE: This is just a proof of concept.
-// Ideally, the wasm api will have more features, like the
-// python api. For now, this is just a sanity check that something
-// works at all.
-//
-// TODO: replace the below with something more robust and feature-filled
+#[wasm_bindgen]
+pub struct Flag;
+
+#[wasm_bindgen]
+impl Flag {
+    #[wasm_bindgen(getter)]
+    pub fn no_neg_div(&self) -> u32 { _no_neg_div }
```

Comment:
technically we don't need this anymore. This was made mandatory in the soft-fork, and it had not been exercised on the chain before the soft-fork.